packages/shared/src/components/modals/Modal.svelte

```diff
@@ -39,38 +48,64 @@
         close()
     }
 
-    let isBlockedByTimeout = false
-    let show = false
+    async function updateMaxHeight(): Promise<void> {
+        await tick()
+        if (!show || !modal) return
```

Comment:
Please take care that we add return to a separate line
```suggestion
        if (!show || !modal) {
            return
        }
```